packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/createproject.md

```diff
@@ -1,154 +1,149 @@
 Launch Xcode and create a new iOS app project using SwiftUI for the interface and Swift for the language.
 
-> **Note:** The sample code is compatible with iOS 14 and later.
+> **Note:** The sample code uses SwiftUI features that are available in iOS 14 and the async/await feature of iOS 15. Use the completion handler version of the async/await functions to add compatibility with iOS 14 and later.
```

Comment:
This makes it sound like only iOS 14+ is supported, when in fact older versions are supported.

```suggestion
> **Note:** The sample code uses SwiftUI features that are available in iOS 14 and the async/await feature of iOS 15. Use the completion handler version of the async/await functions to add compatibility with iOS 14 and earlier.
```

Reply:
I think we had this same exchange before. Although the SDK works with earlier versions, this sample won't. It uses features of SwiftUI that are iOS 14 or later, which is why the initial part of the sentence is there. If that's not clear then I need to revise the second sentence... how about:

>  Use the completion handler version of the async/await functions to enable the sample to run on iOS 14 and later.

Reply:
I like that better, it makes it clearer that it's talking about the sample app itself.